numba/cuda/tests/data/jitlink.cu

```diff
@@ -0,0 +1,20 @@
+// Compile with:
+//
+//   nvcc -gencode arch=compute_53,code=compute_53 -rdc true -ptx jitlink.cu
+//
+// using the oldest supported toolkit version (10.2 at the time of writing).
+
```

Comment:
Suggest adding a note in here about the necessity of the `out` argument due to CUDAs minimal CC.

Reply:
Will do.

Reply:
Done.